docs/user-guide/share.md

```diff
@@ -32,6 +34,18 @@ The **Social** section allows the user to share the resource on the most common
 
 <img src="../img/share/social.jpg" class="ms-docimage"  style="max-width:400px;"/>
 
+## Permalink
+
+The **Permalink** section allows to save the current overall viewer state of the resource and share it as a permalink.
+
+<img src="../img/share/permalink.jpg" class="ms-docimage"  style="max-width:400px;"/>
+
+To obtain the permalink, the user must enter the resource **Title** and **Description** (not mandatory) and choose whether the resource will be public or not by enabling/disabling the **Public** option.
```

Comment:
```suggestion
A permalink is a new resource belonging to a dedicate category in MapStore for which the user must enter the **Title** and **Description** (the last one is not mandatory) and choose whether the resource will be public checking the **Public** option (this will generate a public map permalink so that everyone can access it).
```

Reply:
@ElenaGallo include also a warning below this paragraph to inform that Map Details as well as other resources connected to a Map, if present, will not be available in the final permalink resource.